arangod/Replication2/StateMachines/Document/DocumentLeaderState.cpp

```diff
@@ -650,14 +650,65 @@ auto DocumentLeaderState::createIndex(
           }
 
           if (auto releaseRes = self->release(logIndex); releaseRes.fail()) {
-            LOG_CTX("a03c7", ERR, self->loggerContext)
+            LOG_CTX("26355", ERR, self->loggerContext)
```

Comment:
Why this LogID change?
Not that it is critical as the log cannot be written yet in production

Reply:
I've had a conflict with another LOG_CTX and I just patched one of them, I forgot which one was first. Are you ok with this change, or should I revert?

Reply:
I am fine with the change :+1: